.github/workflows/build-docs.yml

```diff
@@ -23,9 +23,9 @@ jobs:
 
             - uses: actions/checkout@v3
 
-            - uses: pnpm/action-setup@v2
+            - uses: pnpm/action-setup@v4
               with:
-                  version: 8
+                  version: 8.2.0
```

Comment:
I'd rather remove it here and always use the version defined in the package.json